### ISSUE TITLE
Error trying to build - jackson-coreutils:jar:2.0 cannot be resolved

### ISSUE THREAD

**Issue by paoloantinori** [Apicurio/apicurio-studio#1834]
```[INFO] apicurio-studio-platforms-quarkus-api .............. FAILURE [  5.386 s]
[INFO] apicurio-studio-platforms-quarkus-ui ............... SKIPPED
[INFO] apicurio-studio-platforms-quarkus-ws ............... SKIPPED
[INFO] apicurio-studio-test ............................... SKIPPED
[INFO] apicurio-studio-test-integration ................... SKIPPED
[INFO] apicurio-studio-test-integration-common ............ SKIPPED
[INFO] apicurio-studio-test-integration-api ............... SKIPPED
[INFO] apicurio-studio-test-integration-ws ................ SKIPPED
[INFO] apicurio-studio-test-integration-arquillian ........ SKIPPED
[INFO] apicurio-studio-distro ............................. SKIPPED
[INFO] apicurio-studio-distro-quickstart .................. SKIPPED
[INFO] apicurio-studio-test-integration-single-node-keycloak SKIPPED
[INFO] apicurio-studio-tools .............................. SKIPPED
[INFO] apicurio-studio-tools-release ...................... SKIPPED
[INFO] apicurio-studio-tools-httpmon ...................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  06:46 min
[INFO] Finished at: 2022-03-06T12:26:00+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project apicurio-studio-platforms-quarkus-api: Could not resolve dependencies for project io.apicurio:apicurio-studio-platforms-quarkus-api:jar:0.2.52-SNAPSHOT: The following artifacts could not be resolved: com.github.java-json-tools.jackson-coreutils:jackson-coreutils:jar:2.0, com.github.java-json-tools.jackson-coreutils:jackson-coreutils-equivalence:jar:2.0: Could not find artifact com.github.java-json-tools.jackson-coreutils:jackson-coreutils:jar:2.0 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR] 
```

**Comment by paoloantinori** [comment]
This seems to be failing too:

```mvn org.apache.maven.plugins:maven-dependency-plugin:3.0.1:get \
 -DrepoUrl=https://repo.maven.apache.org/maven2 \
 -DgroupId=com.github.java-json-tools.jackson-coreutils \
 -DartifactId=jackson-coreutils \
 -Dversion=2.0 \
 -Dpackaging=jar
```


is that artifact supposed to come from an alternative repo that is not enabled by default?

**Comment by EricWittmann** [comment]
You are getting a weird version of `jackson-coreutils` for some reason.  Can you please run the following from within the `apicurio-studio/platforms/quarkus/api` module?

```
git clone git@github.com:Apicurio/apicurio-studio.git
cd apicurio-studio/platforms/quarkus/api
mvn dependency:tree -Dincludes=com.github.java-json-tools*
```

Here is what I get:

```
$ mvn dependency:tree -Dincludes=com.github.java-json-tools*
[INFO] Scanning for projects...
[INFO]
[INFO] ---------< io.apicurio:apicurio-studio-platforms-quarkus-api >----------
[INFO] Building apicurio-studio-platforms-quarkus-api 0.2.52-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-dependency-plugin:3.2.0:tree (default-cli) @ apicurio-studio-platforms-quarkus-api ---
[INFO] io.apicurio:apicurio-studio-platforms-quarkus-api:jar:0.2.52-SNAPSHOT
[INFO] \- io.quarkus:quarkus-resteasy-jackson:jar:1.13.7.Final:compile
[INFO]    \- org.jboss.resteasy:resteasy-jackson2-provider:jar:4.5.12.Final:compile
[INFO]       \- com.github.java-json-tools:json-patch:jar:1.13:compile
[INFO]          +- com.github.java-json-tools:msg-simple:jar:1.2:compile
[INFO]          |  \- com.github.java-json-tools:btf:jar:1.3:compile
[INFO]          \- com.github.java-json-tools:jackson-coreutils:jar:2.0:compile
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.878 s
[INFO] Finished at: 2022-03-10T14:08:52-05:00
[INFO] ------------------------------------------------------------------------
```

Notice that mine is `com.github.java-json-tools:jackson-coreutils:jar:2.0:compile` but your error message seems to indicate `com.github.java-json-tools.jackson-coreutils:jackson-coreutils:jar:2.0`.

It's subtle, but notice that the groupId is different:

Mine: `com.github.java-json-tools`
Yours: `com.github.java-json-tools.jackson-coreutils`

I'm open to ideas about how this could be the case!

@Apicurio/developers any ideas?

**Comment by paoloantinori** [comment]
Thank you for looking into this. The artifact name issue could be something that I have introduced filing the issue and not a real inconsistency, but thanks for pointing that out.

Additionally, I think we can close this.

I have created a brand new local maven repo, and the build showed no error at all:

```
TMP_MAVEN=$(mktemp -d)
echo $TMP_MAVEN
mvn clean install -Dmaven.repo.local=$TMP_MAVEN
```

So I performed `rm -rf ~/.m2/repository/com/github/java-json-tools/jackson-coreutils` and the error is gone. Not sure what it was. I have double checked file ownership, in case that was the issue but everything was owned by my user. Gremlin.

Thanks for looking into this!
